src/plugins/kibana/public/management/index.js

```diff
@@ -42,6 +42,16 @@ uiModules
       if ($scope.section) {
         $scope.section.items.forEach(item => {
           item.class = `#${$location.path()}`.indexOf(item.url) > -1 ? 'active' : undefined;
+          // Disable link if path is not set
+          if (!item.path) {
+            delete item.url;
+          }
+          item.items.forEach(childItem => {
+            // Disable link if path is not set
+            if (!childItem.path) {
+              delete childItem.url;
```

Comment:
Looks like that's possible. Given that the `url` is constructed from the `path` (if `url` isn't explicitly specified) in the `ManagementSection` class, maybe the right approach here would be to define a method in `ManagementSection` class like `isLinkEnabled` and use that here?